lib/AuditoManager.dart

```diff
@@ -52,53 +25,37 @@ class AudioManager {
         isPlaying = false;
       }
     });
-    print("Playing audio: $audioFile");
+    debugPrint("Playing audio: $audioFile");
```

Comment:
Still waiting to see 1. from [this comment](https://github.com/Kenneth-W-Chen/mindfulness-app/pull/16#discussion_r1827020349) to be added. Tell me it's impossible if it is.

Reply:
Still not resolved